devito/finite_differences/derivative.py

```diff
@@ -207,6 +206,30 @@ def evaluate(self):
             res = generic_derivative(expr, *self.dims, self.fd_order,
                                      self.deriv_order, stagger=self.stagger[0],
                                      matvec=self.transpose)
-        for e in self._eval_at:
-            res = res.xreplace(e)
         return res
+
+
+class Derivative_at(Derivative):
```

Comment:
Never use _ in class names

I **think** this can be made general (``At`` or ``Eval``), ie not specifically related to Derivative

Reply:
Sure